prov/verbs/src/verbs_cq.c

```diff
@@ -230,47 +230,184 @@ static void fi_ibv_cq_read_data_entry(struct ibv_wc *wc, int i, void *buf)
 		wc->byte_len : 0;
 }
 
-static int fi_ibv_match_ep_id(struct slist_entry *entry, const void *ep_id)
+static inline int fi_ibv_match_ep_id(struct slist_entry *entry,
```

Comment:
nit: This function is passed as a callback parameter.  It can't be inlined.

Reply:
Not follow why I've added `inline` to the function.
Thanks!